testSuite/testCommonFunctions/testCommonFunctions.m

```diff
@@ -573,29 +518,28 @@ function resolution_polly_paraPoints(testCase)
 
             outputs = resolution_polly_paraPoints(testCase.resolution_polly_paraPointsInputs{1:end});
 
-            testCase.verifyEqual(testCase.resolution_polly_paraPointsOutputs,outputs);
-
-        end
-  %}       
-%         function resolution_polly_test(testCase)
-%             % test resolution_polly
-%             %
-%             % Inputs:
-%             %   testCase - class instance
-%             %
-%             % Outputs:
-%             %   none
-%             %
-%             % Example:
-%             %   resolution_polly(testCase)
-%             
-%             out1 = resolution_polly(testCase.resolution_pollyInputs{1:end});
-%             outputs ={out1};
-% 
-%             testCase.verifyEqual(testCase.resolution_pollyOutputs,outputs);
-% 
-%         end
-%        
+            testCase.verifyEqual(testCase.resolution_polly_paraPointsOutputs,outputs, "RelTol", testCase.tolerance, "AbsTol", testCase.abs_tolerance);
+
+        end
+ %}    
+
+        function testResolutionPolly(testCase)
+            % testResolutionPolly
```

Comment:
Docstring's don't give useful information in tests. Please remove and you can add a comment that gives more information about the test if needed. Many of these test don't even need any comment since the test methods are named properly